compiler/src/iree/compiler/DispatchCreation/Passes.cpp

```diff
@@ -221,34 +221,37 @@ static void addDispatchRegionCreationPasses(OpPassManager &passManager) {
       .addPass(DispatchCreation::createFormScalarDispatchesPass)
       // Create `flow.dispatch.region` centered around a root and fuse with
       // producers and consumers.
-      .addPass([&]() {
+      .addPass([] {
```

Comment:
I dont know how `static global` values are supposed to interact with the no-capture here.

Reply:
Capture is only for local variables

Reply:
https://en.cppreference.com/w/cpp/language/lambda#:~:text=A%20lambda%20expression%20can%20use%20a%20variable%20without%20capturing%20it%20if%20the%20variable